Added ElevenLabs to Microsoft Teams Message Sample

cassxw opened the pull request:
- For one of the demos for the ElevenLabs webinar.

cassxw committed:
Create ElevenLabsConversationalAItoMicrosoftTeamsMessage.json

TerrenceMeiring commented:
The logic behind the workflow is very good.

There are some features I think we should add to assist the client:

The description explains fast follow up, but the timer only runs once a day. Suggest either updating the description to explain that it is a once a day report or update the workflow to run regularly.

Would recommend duplication prevention. 
<img width="544" height="375" alt="{16D6EB95-4D7B-455B-A96E-8DBCE0D3647B}" src="https://github.com/user-attachments/assets/53468c34-899b-44ae-9803-4b5a5d7bd433" />
Add a reduce node between these two, followed by an if node to see if there is any new data coming out. remember to add the commit at the end.

TerrenceMeiring commented on workflow-templates/ElevenLabsConversationalAItoMicrosoftTeamsMessage.json:
Please see conversation for requested changes.

cassxw committed:
Update ElevenLabsConversationalAItoMicrosoftTeamsMessage.json

cassxw commented:
@TerrenceMeiring - Thank you so much for your review!

After discussion with Daniel this morning, I've opted to make this Workflow for the demo be a bit cleaner. Updated the description, used a "choose" node, and removed some formatting.

We also decided to remove the DailyProductDigest aspect into a separate Workflow, which I added your suggested Reduce mechanism there. If you can please check that I implemented that correctly, just in case we do show that in the webinar: https://app.flowgear.net/#t-behlruij/sites/32115adf-b283-42da-853a-46a0719581bc/workflows/90ef4cdf-eeb4-44c5-903b-301fad881f95/design

But, now, the updated Workflow above is just a lightweight alerting system, into 3 teams channels. This will be the main demo, just to show off the post-call analysis with the ElevenLabs Conversational AI connector.

cassxw committed:
Removed stray Connections on Teams nodes
